AUTHORS

```diff
@@ -1102,16 +1102,14 @@ Michael-Rainabba Richardson <rainabba@gmail.com>
 oogz <oogz@users.noreply.github.com>
 Rene Weber <weber.rene@live.de>
 Lauren Spiegel <lhspiegel@gmail.com>
-Lydia Katsamberis <llkats@gmail.com>
+Lydia Kats <llkats@gmail.com>
```

Comment:
I assume the first one is a more formal name?

Reply:
I chose the one that is on their GitHub profile: https://github.com/llkats